packages/framework/tinylicious-client/src/test/DiceRoller.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DataObject, DataObjectFactory, IDataObjectProps } from "@fluidframework/aqueduct";
+import { IEvent } from "@fluidframework/common-definitions";
+
+export class DiceRoller extends DataObject {
```

Comment:
Great idea -- since it's no longer a dice roller though, would you please rename to something like `TestDataObject`?

Reply:
@ChumpChief Done.